lagom-client-restaurant/menu-item-impl/deploy/kubernetes/menu-item-deployment.yaml

```diff
@@ -37,14 +35,8 @@
          - configMapRef:
              name: menu-item-config
          env:
-           - name: "HTTP_BIND_ADDRESS"
-             value: "0.0.0.0"
```

Comment:
It isn't necessary to override the bind address. Play and Lagom use 0.0.0.0 by default.